infra/gp-multena/templates/03-deployment.yaml

```diff
@@ -0,0 +1,118 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "gp-multena.name" . }}-proxy
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "gp-multena.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "gp-multena.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "gp-multena.selectorLabels" . | nindent 8 }}
+    spec:
+      securityContext:
+        runAsNonRoot: true
+        seccompProfile:
+          type: RuntimeDefault
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- if .Values.serviceAccount.create }}
+      serviceAccountName: {{ include "gp-multena.name" . }}
+      {{- end }}
+      containers:
+        - name: multena-proxy
+          securityContext:
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop: [ "ALL" ]
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default "main" }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: http
+              containerPort: 8080
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: http
+          readinessProbe:
+            httpGet:
+              path: /healthz
+              port: http
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
+          volumeMounts:
+            - mountPath: /etc/ssl/certs/
+              name: openshift-service-ca
+            - mountPath: /etc/config/config/
+              name: multena-config
+              {{- if eq .Values.multena.provider "configmap" }}
+            - mountPath: /etc/config/users/
+              name: multena-users
+            - mountPath: /etc/config/groups/
+              name: multena-groups
+              {{- end }}
+              {{- if eq .Values.multena.provider "mysql" }}
+            - mountPath: /etc/secret/db/
+              name: multena-db-secret
+              {{- end }}
+        - name: prom-label-proxy
+          securityContext:
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop: [ "ALL" ]
+          args:
+            - --insecure-listen-address=0.0.0.0:9095
+            - --upstream={{ .Values.multena.thanosUrl }}
+            - --label={{ .Values.multena.tenantLabel }}
+            - --query-param={{ .Values.multena.tenantLabel }}
+            - --enable-label-apis
+            - --error-on-replace
+          image: ghcr.io/lucostus/prom-label-proxy:multi-value-regex-v0.0.4
```

Comment:
bitte in ein gepardec/gepaplexx tag ändern. 
image konfigurierbar machen über values und entsprechende defaults im values.yaml setzen. 
semVer bei tag beachten. 

gibt es kompatibilitätsprobleme zwischen prom-label-proxy und multena proxy?  wenn ja, sind die wo dokumentiert?